src/img-view.cc

```diff
@@ -1594,13 +1594,13 @@ static GtkWidget *view_confirm_dir_list(ViewWindow *vw, GList *list)
 	g_signal_connect(G_OBJECT(menu), "destroy",
 			 G_CALLBACK(view_dir_list_destroy), d);
 
-	menu_item_add_stock(menu, _("Dropped list includes folders."), GTK_STOCK_DND_MULTIPLE, nullptr, nullptr);
+	menu_item_add_stock(menu, _("Dropped list includes folders."), GQ_ICON_DND, nullptr, nullptr);
```

Comment:
```suggestion
	menu_item_add_icon(menu, _("Dropped list includes folders."), GQ_ICON_DND, nullptr, nullptr);
```
or is it really have to be `menu_item_add_stock`?

Reply:
For now it needs to stay *_stock since I didn't find a correct replacement for `GTK_STOCK_DND_MULTIPLE` and `GQ_ICON_DND` resolves to that. Similar situation is with `GQ_ICON_COLLECTION`.